src/features/MenuPage/MenuCourses/Refreshments/Refreshments.js

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from "react";
+import MenuItem from "../MenuItem";
+
+const Refreshments = () => {
+
+    const [refreshments, setRefreshments] = useState([])
+
+    const getRefreshments = async () => {
+        const data = await fetch('http://localhost:3001/dishes/refreshments')
+        return await data.json()
+    }
+
+    useEffect(() => {
+        getRefreshments()
+            .then(json => {
+                if (json.success) {
+                    setRefreshments(json.data)
+                }
+            })
+    },[])
+
+    const displayRefreshments = () => {
+        if (refreshments.length > 0) {
+            return refreshments.map(refreshment => {
+                return <MenuItem key={refreshment._id} name={refreshment.name} description={refreshment.description}
+                                 price={refreshment.price} image={refreshment.imageURL}/>
+            })
+        }
+        else return null
```

Comment:
Would be nice to show an error message here - maybe some kind of generic error component that you can populate with a specific message